packages/bloc/lib/src/bloc.dart

```diff
@@ -157,15 +157,20 @@ abstract class Bloc<Event, State> extends Stream<State> implements Sink<Event> {
   /// ```
   Stream<State> transformStates(Stream<State> states) => states;
 
+  Stream<State> _buildStateStream() async* {
+    yield _state;
+    yield* _stateController.stream;
+  }
+
   void _bindStateSubject() {
```

Comment:
can we rename this `_bindEventsToStates` since we're no longer using subjects?

Reply:
Done